ingest/cell_metadata.py

```diff
@@ -238,11 +256,11 @@ def validate_type_annotations(self):
         invalid_types = []
         # skipping the TYPE keyword, iterate through the types
         # collecting invalid type annotations in list annots
-        for t in self.metadata_types[1:]:
-            if t not in self.annotation_type:
+        for t in self.annot_types[1:]:
+            if t.lower() not in ('group', 'numeric'):
                 # if the value is a blank space, store a higher visibility
                 # string for error reporting
-                if not t:
+                if 'unnamed' in t:
```

Comment:
Thanks for explaining 'Unnamed' as a special Pandas value in our chat, which could have more characters in the string than just 7 characters in 'Unnamed'.  So the current use of `in` seems best. 

However, we agree the letter casing bug remains.  So this would update to either:

```suggestion
                if 'unnamed' in t.lower():
```

or 

```suggestion
                if 'Unnamed' in t:
```

(The [`lower()`](https://docs.python.org/3/library/stdtypes.html#str.lower) method doesn't mutate string variables it operates on.)

Reply:
9a2b943

Reply:
Thank you!  Approving now.